can_bus/quadruna/Debug/Debug_tx.json

```diff
@@ -121,5 +121,21 @@
                 "bits": 1
             }
         }
+    },
+    "CellDiagnostics": {
+        "msg_id": 707,
+        "cycle_time": 1000,
+        "description": "Signals related to BMS cell voltage and temperature diagnostics.",
+        "signals": {
+            "BMSDiagnosticsRequest": {
+                "bits": 1
+            },
+            "SegmentCellVoltageRequest": {
+                "enum": "SegmentCellVoltageRequest"
+            },
+            "SegmentTemperaturesRequest": {
+                "bits": 1
+            }
```

Comment:
is the idea here to choose which segment specifically you want to debug and only show information about that segment? If so, all messages are currently configured to broadcast all the time (as per BMS_tx.json)